docs/guides/gcp-pcf/director.yml.erb

```diff
@@ -24,6 +24,9 @@ cf-tcp-router-name: <%= ENV['tcp_router_domain'] %> # Hostname of TCP router. Us
 routing-cf-api-url: https://api.<%= ENV['cf_system_domain'] %> # TCP routing API URL. In PCF: https://api.<system domain>
 routing-cf-client-id: <%= ENV['routing_cf_client_id'] %> # Routing UAA client name that will be used to register tcp routes
 routing-cf-uaa-url: https://uaa.<%= ENV['cf_system_domain'] %> # Routing UAA URL In PCF: https://uaa.<system domain>
+routing-cf-app-domain-name: <%= ENV['cf_apps_domain'] %> # Cloud Foundry apps domain
+routing_cf_nats_internal_ip: <%= ENV['cf_nats_internal_ip'] %> # Internal IP address of Cloud Foundry NATS
```

Comment:
can you s/routing_cf_nats_internal_ip/routing-cf-nats-internal-ip/g for consistency?

besides the times we make it an ENV var (unfortunately)